portality/formcontext/formcontext.py

```diff
@@ -1056,15 +1052,11 @@ def finalise(self):
         # inform publisher if this was set to 'in progress' from 'pending'
         if self.source.application_status == constants.APPLICATION_STATUS_PENDING and self.target.application_status == constants.APPLICATION_STATUS_IN_PROGRESS:
             if app.config.get("ENABLE_PUBLISHER_EMAIL", False):
-                try:
-                    emails.send_publisher_inprogress_email(self.target)
-                    self.add_alert('An email has been sent to the Journal Contact alerting them that you are working on their application.')
-                except app_email.EmailException:
-                    magic = str(uuid.uuid1())
-                    self.add_alert('Hm, sending the ready status to publisher email didn\'t work. Please quote this magic number when reporting the issue: ' + magic + ' . Thank you!')
-                    app.logger.exception('Error sending ready status email to publisher - ' + magic)
+                alerts = emails.send_publisher_inprogress_email(self.target)
+                for alert in alerts:
+                    self.add_alert(alert)
             else:
-                self.add_alert('Did not send email to Journal Contact about the status change, as publisher emails are disabled.')
+                self.add_alert('Did not send email to Owner or Journal Contact about the status change, as publisher emails are disabled.')
```

Comment:
should this be a Message?

Reply:
Yeah, should be.  There's a lot in that file that should be, but I'll stick to only moving stuff out as we touch it.  So, I've moved this one :)